### ISSUE TITLE
Can't remove site tags

### ISSUE THREAD

**Issue by ahuanguchi** [neocities/neocities#170]
I don't know if this is happening to anyone else, but I'm unable to remove any of my site's tags except the most recently added tag no matter how many I select (I have 5 tags). I can select them and click on the "Remove Tag(s)" button, but nothing happens.

**Comment by ahuanguchi** [comment]
Now I'm unable to remove any tags. Also, I'm not sure if this is related, but my site (d-m-adventure) currently isn't searchable by its tags. Again, I don't know if this is a general issue or something that only affects a few sites.

**Comment by kyledrake** [comment]
This has been a finicky function in the past for some reason. At the moment, I am not able to reproduce this. Could you provide me with your site username so I can take a look? (Feel free to email it if you'd prefer not to make it public).

**Comment by ahuanguchi** [comment]
My site is d-m-adventure. Would adding screenshots of the bug help, or is this something you've been able to reproduce in the past?

**Comment by kyledrake** [comment]
@ahuanguchi I'm still not sure what's going on here. Could you see if it's still an issue? If it is, I can make the changes you'd like manually. Sorry about the trouble.

**Comment by ahuanguchi** [comment]
@kyledrake I just tried again, and I still can't remove tags on my profile page or activity page using Chrome or Firefox. I think something's going wrong when the post request to https://neocities.org/tags/remove is made. The form data gets sent with `tags[]:fun`, I get a status code of 302 Found, and my browser returns to the same page, but nothing changes. There isn't necessarily anything I need to change right now, but I figured this was something I should share.

Before:
![before](https://cloud.githubusercontent.com/assets/10644362/8839398/97820728-309a-11e5-8b27-e9a936fd2036.png)
After:
![after](https://cloud.githubusercontent.com/assets/10644362/8839399/97901c0a-309a-11e5-8d92-a44258c28238.png)
